pkg/operator/controller/ingress/deployment.go

```diff
@@ -421,6 +424,14 @@ func desiredRouterDeployment(ci *operatorv1.IngressController, ingressController
 		Name:  RouterLoadBalancingAlgorithmEnvName,
 		Value: loadBalancingAlgorithm,
 	})
+	reloadInterval := 5
+	if unsupportedConfigOverrides.ReloadInterval != 0 {
```

Comment:
```suggestion
	if unsupportedConfigOverrides.ReloadInterval > 0 {
```

Might as well avoid negative numbers, right?

Reply:
Fixed.  Thanks!